com.unity.render-pipelines.universal/Shaders/SpatialMappingWireframe.shader

```diff
@@ -0,0 +1,153 @@
+Shader "Universal Render Pipeline/SpatialMappingWireframe"
```

Comment:
Change to `Universal Render Pipeline/VR/SpatialMapping/Wireframe` for better consistency with existing shader